README.md

```diff
@@ -1,8 +1,26 @@
-# ESP-BSP
-Board support packages for Espressif development boards.
+# ESP-BSP: Espressif's Board Support Packages
+Board support packages for development boards using Espressif's SoCs, written in C.
+
+## Supported boards
```

Comment:
Probably links to particular Boards datasheets can be added. It can be just under Board name.

Just a small graphic point, maybe we can consider similar style of pictures. eg. all in PNG without background/all with white background